stories/Button.story.tsx

```diff
@@ -1,27 +1,33 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { Button, Icon } from "../src/index";
+import { ButtonHeights } from "../src/Buttons/button.style";
```

Comment:
No changes for this PR but for the future I think its good to keep in mind the interfaces of the components, and the usability. I would here be better for button to either export button hights from the component file itself, or to expect one of a number of strings, so we do not need to have additional imports. I think the value of hiding the true value behind a variable here is less than an easy to use component. This was then also kind of a breaking change as we can see by this story, so the previous release should have been a 3.0

Reply:
I don't think it breaking because I used the same value after all, it's just a TS thing which only affects TS not js consumers I hope
I will move ButtonHeights into the component